articles/api-auth/using-rules/_includes/_runtime.md

```diff
@@ -0,0 +1,10 @@
+The Auth0 Runtime expects you to return an `access_token` that looks like the following:
```

Comment:
Here we need to update this for the new response e.g. `{accessToken: myCustomAccessTokenClaims, idToken: myCustomIdTokenClaims}`